inputgraph/loader.go

```diff
@@ -367,9 +367,7 @@ func (l *loader) handleImport(ctx context.Context, cmd spec.Command) error {
 		alias = cmd.Args[2]
 	}
 
-	isGlobal := l.target.Target == "base"
-
-	err := l.varCollection.Imports().Add(cmd.Args[0], alias, isGlobal, false, false)
+	err := l.varCollection.Imports().Add(cmd.Args[0], alias, false, false, false)
```

Comment:
Once I fixed the above issue, the use of global imports here led to duplicate errors. I'm not 100% clear on how global imports are meant to behave. Are they global to a file or global across the entire execution context (all files)?

Reply:
Global imports are global within the file only.